s3tests_boto3/functional/test_s3select.py

```diff
@@ -388,15 +388,22 @@ def create_list_of_int(column_pos,obj,field_split=",",row_split="\n"):
 
     return list_of_int
 
+def get_max_from_parquet_column(parquet_obj, column_name):
+    table = pq.read_table(pa.BufferReader(parquet_obj))
+    
+    df = table.to_pandas()
+    
+    return df[column_name].max()
+
 @pytest.mark.s3select
 def test_count_operation():
     csv_obj_name = get_random_string()
     bucket_name = get_new_bucket_name()
     num_of_rows = 1234
     obj_to_load = create_random_csv_object(num_of_rows,10)
     upload_object(bucket_name,csv_obj_name,obj_to_load)
-    res = remove_xml_tags_from_result( run_s3select(bucket_name,csv_obj_name,"select count(0) from s3object;") ).replace(",","")
-
+    res = remove_xml_tags_from_result( run_s3select(bucket_name,csv_obj_name,"select count(0) from s3object;","CSV") ).replace(",","")
+    
```

Comment:
the "CSV" parameter is default. 
no?

Reply:
Yes, default is CSV. Don't need to override the same. Will remove it.